libweb3jsonrpc/Web3.h

```diff
@@ -9,7 +9,7 @@ namespace rpc
 class Web3: public Web3Face
 {
 public:
-	Web3(std::string _clientVersion = "C++ (ethereum-cpp)"): m_clientVersion(_clientVersion) {}
+	Web3(const std::string& _clientVersion = "C++ (ethereum-cpp)"): m_clientVersion(_clientVersion) {}
```

Comment:
Here pass by value should stay, but `std::move` may be used on the initialization list. Can you also change the default value to `cpp-ethereum`?